hbase-resource-bundle/src/main/resources/supplemental-models.xml

```diff
@@ -977,6 +977,24 @@ Copyright 2010 FasterXML.com
     </project>
   </supplement>
 
+  <supplement>
+    <project> <!-- hadoop.profile=3.0 from hadoop-3.3.0 -->
+      <groupId>org.jline</groupId>
+      <artifactId>jline</artifactId>
+      <version>3.9.0</version>
+      <licenses>
+        <license>
+          <name>BSD license</name>
+          <url>https://opensource.org/licenses/BSD-3-Clause</url>
+          <distribution>repo</distribution>
+          <comments>
+            Copyright (c) 2002-2018, the original author or authors.
```

Comment:
just wondering, does it matter that we are in 2019 now?

Reply:
not unless the version of jline we end up packaging is from 2019.